index.html

```diff
@@ -259,6 +269,36 @@
         <div class="prepend-input">💣</div>
         <input id="bombs" class="input" type="number" value="10">
 
+        <select id="theme" onchange="_change(this);" class ="box">
```

Comment:
This event should be bound in https://github.com/muan/emoji-minesweeper/blob/gh-pages/game.js#L55 like the others.